lib/settings.rb

```diff
@@ -16,16 +16,131 @@
 module Settings
   module Db
     class Store < RailsSettings::Base
-      source Rails.root.join("config/app.yml")
-      namespace Rails.env
-
-      # Workaround (this method missing in current implementation, but RailsSettings-UI
-      # expects it -- doesn't really need to use it any more b/c RailsSettingsCached
-      # deals with defaults internally)
-      def self.defaults
-        result = RailsSettings::Default.enabled? ? RailsSettings::Default.instance : {}
-        result.with_indifferent_access
-      end
+      field :push_salesforce_lead_enabled, type: :boolean, default: false
+      field :user_info_error_emails_enabled, type: :boolean, default: false
+      field :send_google_analytics, type: :boolean, default: false
+      field :google_analytics_code, type: :string, default: 'UA-73668038-2'
+      field :google_tag_manager_code, type: :string, default: 'GTM-W6N7PB'
+      field :subjects, type: :hash, default: {
```

Comment:
I don't think these are used anymore - the books get fetched from the CMS... but we can save removing this for another PR.